src/components/atoms/SortDropdown.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="select-wrapper">
+    <select>
+      <option v-for="(option, index) in options" :key="index">
+        {{ option }}
+      </option>
+    </select>
+    <font-awesome-icon :icon="dropdownIcon" class="arrow-icon" />
+  </div>
+</template>
+
+<script>
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import { COLORS } from '../../../public/Consts';
+
+export default {
+  name: "SortDropdown",
+  components: {
+    FontAwesomeIcon
+  },
+  props: {
+    options: {
```

Comment:
Hardcode it with [ "najbliższe", "najdalsze", "najnowsze", "najstarsze", "najwyżej oceniane"] with "najbliższe" as default